src/server-manager/cache-manager.ts

```diff
@@ -94,7 +115,8 @@ export class CacheManager {
     if (!this.cacheImageMessageRawPayload) {
       throw new Error(`${PRE} getMessage() has no cache.`)
     }
-    return this.cacheImageMessageRawPayload.get(messageId)
+    const cacheData = await this.cacheImageMessageRawPayload.get(messageId)
+    return cacheData ? cacheToPadplusMessagePayload(cacheData) : undefined
```

Comment:
if `cacheData` does not exist, it will throw error in `cacheToPadplusMessagePayload()`, so let's put this logic into `cacheToPadplusMessagePayload()`?

Reply:
I think your style is better. So just ignore this comment.